config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
 
+  get 'goals/create'
```

Comment:
No, `rails g` must have made this too. I deleted the specs associated with it, but missed the route itself.